test/scapy/layers/bluetooth.uts

```diff
@@ -408,6 +408,20 @@ assert evt_pkt[HCI_LE_Meta_Connection_Update_Complete].timeout == 60
 
 
 + Bluetooth LE Advertising / Scan Response Data Parsing
+= Parse EIR_CompleteList32BitServiceUUIDs
```

Comment:
Could you provide tests for some of the others?

Reply:
It will possible take me a while. I will try to collect samples that I can share. Let's convert this to a draft meanwhile.